.jenkins/pace_physics_cache_setup.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [[ "${NODE_NAME}" == *"daint"* ]] ; then source ~/.bashrc ; fi
+set -e
+export LONG_EXECUTION=1
+.jenkins/jenkins.sh run_physics_regression_tests ${backend} ${experiment}
+cd buildenv && git reset --hard && cd ..
+.jenkins/jenkins.sh run_physics_parallel_regression_tests ${backend} ${experiment}
+.jenkins/generate_caches.sh ${backend} ${experiment}
```

Comment:
Does it make more sense to have pace_physics_cache_setup.sh call jenkins.sh twice, or to make an action that calls these two other actions and then generate_caches.sh, which is called by jenkins.sh once?

Reply:
I did try to make a new action that calls both tests, but was getting the same issue with buildenv (and maybe other folders?) being in a dirty state after the serial test. I think this is fine unless there's a preference to get a new action for running both tests working?

Reply:
The downside of this mode is a lot of logic in jenkins.sh will run twice, making the plan take longer. Since this isn't a PR plan, that's probably not an issue this time. I'm more concerned that there's an issue with these plans where you can't run scripts consecutively because buildenv is left in a dirty state. Your design is being dictated by that behavior, instead of by what you think makes the most sense. Ideally our scripts shouldn't modify buildenv, and should copy any files they need to modify.

But that's outside the scope of this PR, you can leave this script as is. Thanks for the explanation!